### PR TITLE
Create loaded

### PR DESCRIPTION
**Pull request by tstruk**
Add support for Create Loaded command

### CODE REVIEW

File: test/integration/create-loaded.int.c [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                                &auth_rsp);
+    if (rc == TPM2_RC_SUCCESS) {
+        LOG_INFO("success object handle: 0x%x", object_handle);
+    } else {
+        LOG_ERROR("CreateLoaded FAILED! Response Code : 0x%x", rc);
+        exit(1);
+    }
+
+    memset(&out_public, '\0', sizeof(out_public));
+    memset(&name, '\0', sizeof(name));
```
Comment (flihp) [inline test/integration/create-loaded.int.c]: can you, or don't you already do this when the structure is initialized?

Comment (tstruk) [reply test/integration/create-loaded.int.c]: I agree that we can have some common init helpers, but in this case it is not redundant.
Both of them will be populated with data by the Tss2_Sys_CreateLoaded(). I want to clean them fresh before calling to Tss2_Sys_ReadPublic()

**Comment by flihp** [comment]
I just tried to use the UI to fixup the conflict #747 caused when it was merged. Looks like it made a merge commit instead of fixing it up into your commit.

**Comment by flihp** [comment]
oh man github is just a writhing pile of merge commits. I'll fix this up manually to get it out of the queue.

**Comment by flihp** [comment]
replaced by #748

**Pull request closed without merging**